integration_test/third_party_apps_data/applications/mongodb/debian_ubuntu/install

```diff
@@ -9,6 +9,12 @@ osname="$(lsb_release -si)"
 
 case $osname in 
   Ubuntu)
+    if [ "$oscode" = "lunar" ]; then
+      # Ubuntu 23.04 is not officially supported, so use 'focal'.
```

Comment:
Our customers aren't going to do that, are they? Should we just disable MongoDB support on Ubuntu 23.04 instead?